x-pack/plugins/ml/public/ui_actions/open_in_anomaly_explorer_action.tsx

```diff
@@ -97,10 +97,13 @@ export function createOpenInExplorerAction(
         });
       } else if (isAnomalyChartsEmbeddableContext(context)) {
         const { embeddable } = context;
-        const { jobIds, entityFields } = embeddable;
+        const { jobIds$, selectedEntities$ } = embeddable;
 
+        // @TODO: remove
```

Comment:
Is this comment still needed?

Reply:
Removed here https://github.com/elastic/kibana/pull/183456/commits/adaa3aa7a3d88f30ac79b12d339c215bef74422c